rover_msgs/CarouselClosedLoopCmd.lcm

```diff
@@ -0,0 +1,6 @@
+package rover_msgs;
+
```

Comment:
why is closedloopcmd and data the same struct? just use one like CarouselPosition and publish it to different channels???

Reply:
sure